examples/ensemble/plot_stack_predictors.py

```diff
@@ -45,7 +45,7 @@
 
 
 def load_ames_housing():
-    df = fetch_openml(name="house_prices", as_frame=True)
+    df = fetch_openml(name="house_prices", as_frame=True, parser="pandas")
```

Comment:
The issue here is how `OrdinalEncoder` handles `None` differently than `nan`. With the `liac-arff` parser, the missing value for "MiscFeature" was `None`, while with the pandas parser the missing values were `nan`.

```python
from sklearn.datasets import fetch_openml
from sklearn.preprocessing import OrdinalEncoder

df_pandas = fetch_openml(name="house_prices", as_frame=True, parser="pandas")
df_arff = fetch_openml(name="house_prices", as_frame=True, parser="liac-arff")

df_pandas['frame']["MiscFeature"].unique()
# array([nan, 'Shed', 'Gar2', 'Othr', 'TenC'], dtype=object)

df_arff['frame']["MiscFeature"].unique()
# array([None, 'Shed', 'Gar2', 'Othr', 'TenC'], dtype=object)

encoder = OrdinalEncoder()

# output nans
encoder.fit_transform(df_pandas['frame'][["MiscFeature"]])

# encodes the None
encoder.fit_transform(df_arff['frame'][["MiscFeature"]])
```

The quick fix is encoding the missing value:

```python
cat_tree_processor = OrdinalEncoder(
    handle_unknown="use_encoded_value", unknown_value=-1, encoded_missing_value=-2
)
```